components/input/navigationRectangle.js

```diff
@@ -1,58 +1,142 @@
-import { StyleSheet, View } from "react-native";
-import { Icon, Text } from "react-native-paper";
+import { View } from "react-native";
+import { Divider, Icon, Text } from "react-native-paper";
 
 import { getIconByKey } from "~/Utility";
 
 export default function NavigationRectangle({
   drillInfo,
-  attemptShots,
   inputValues,
-  shotIndex,
+  shot,
+  currentShot,
 }) {
+  // console.log("inputValues", inputValues);
+  const keys = Object.keys(inputValues);
   return (
-    <View style={styles.container}>
-      <View style={styles.rowContainer}>
-        <Text>
-          Shot {shotIndex}/{attemptShots.length}
-        </Text>
-        <Text>
-          Target: {attemptShots[shotIndex - 1].target}
-          {drillInfo.requirements[0].distanceMeasure}
-        </Text>
-      </View>
-      <View style={styles.rowContainer}>
-        {drillInfo.inputs.map((item, id) => (
-          <View style={styles.horizontalContainer} key={id}>
-            <Icon source={getIconByKey(item.id)} />
-            <Text>
-              {inputValues[item.id]} {item.distanceMeasure}
-            </Text>
+    <View
+      style={{
+        shadowColor: "black",
+        shadowOffset: {
+          width: 0,
+          height: 3,
+        },
+        backgroundColor: "transparent",
+        shadowOpacity: 0.2,
+        shadowRadius: 5,
+        padding: 10,
+        paddingLeft: 20,
+        paddingRight: 20,
+        maxHeight: 250,
+        width: "90%",
+        overflow: "hidden",
+        flexDirection: "row",
+        alignItems: "center",
+        justifyContent: "space-between",
+      }}
+    >
+      <View
```

Comment:
I tried for a bit to remove this View because it's just a View inside another View, but I can't quite replicate what you have right now... It's fine to leave it, but convention would say you prob shouldn't have a View's child only be another View. Like if you can merge it somehow.

Reply:
This was the only way I could find to have the drop shadow be for the container and not the children elements. It is a little whack for sure but it was the only way I was able to get it to work.

Reply:
ok

Reply:
I managed to do it lol

Reply:
do you wanna figure it out or do you want me to push my changes?